mindboggle/mindboggle123

```diff
@@ -87,6 +87,9 @@ adv_args.add_argument("--fs_openmp", dest="openmp",
 adv_args.add_argument("--fs_T2image", dest="T2image",
                       type=str,
                       help="Optional T2 image to use with FreeSurfer")
+adv_args.add_argument("--fs_3T", dest="T3",
```

Comment:
would it make sense to make a more generic string argument here and pass any of those flags to recon-all? the T2 image one was added to help with bids support and that it needed an input image.